cypress/integration/team/teammates_pagination_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const uuid = () => Cypress._.random(0, 1e6);
+
+describe('Teams Suite', () => {
+    before(() => {
+        // MaxUsersPerTeam setting must be >60.
+        // Requires more than 60 available test users.
+        // (e.g., make cypress-test-data)
+        Cypress.env('team-size', 60);
+
+        // 1. New test case user
```

Comment:
nit: Just recently, we decided not to add number for test step.  We just use `#` instead.
```suggestion
        // # New test case user
```

Reply:
I used all #  in the today's PR.